SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Qiskit.org only  supports the online version at a time both for bug, which covers `https://*.qiskit.org/*`.
```

Comment:
I don't understand what the "both for bug" part means?

Reply:
Same here